index.html

```diff
@@ -152,7 +152,10 @@ <h2 class='shadow-text-strong'>{{ slide.title.text }}</h2>
   </div>
 
   <div class="row modCallToAction centered-text">
-    <p>Die Werkstatt hat <b>Donnerstags 18-21 Uhr</b> und <b>Sonntags 15-19 Uhr</b> geöffnet.</p>
+    <p>
```

Comment:
Isn't it ok, if you still leave the first part of the initial phrase in there `Die Die Werkstatt hat` ?

Reply:
@simonamoise will you take care of it wenn you're back from Nürnberg? I'll merge it now, because people are already complaining on facebook 😅